pkg/cri/store/image/image.go

```diff
@@ -132,23 +131,20 @@ func getImage(ctx context.Context, i containerd.Image) (*Image, error) {
 	if err != nil {
 		return nil, fmt.Errorf("get image config descriptor: %w", err)
 	}
+
 	id := desc.Digest.String()
 
-	rb, err := content.ReadBlob(ctx, i.ContentStore(), desc)
+	spec, err := i.Spec(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("read image config from content store: %w", err)
-	}
-	var ociimage imagespec.Image
-	if err := json.Unmarshal(rb, &ociimage); err != nil {
-		return nil, fmt.Errorf("unmarshal image config %s: %w", rb, err)
+		return nil, fmt.Errorf("failed to read get image spec: %w", err)
```

Comment:
nit: error message should probably be "failed to retrieve image spec" (or remove one of "read" or "get") :)

Reply:
the error is already decorated.. just return it I think...

Reply:
@estesp updated wording.

@mikebrow I think it worth to keep it to emphasize that error happened when trying to get OCI image spec. The underlying error may be too generic (like when unmarhsaling json). WDT?